graphics/ogre/Portfile

```diff
@@ -27,16 +28,19 @@ homepage            https://www.ogre3d.org/
 platforms           darwin
 supported_archs     x86_64
 
-checksums           rmd160  f0bc5db67c8487ab25afb5f48641fc793460bddc \
-                    sha1    8a1da39713cfefbe0baa4e39095214781506ff6b \
-                    sha256  64e24cc00b94c884988dc1d92b1225abf7553cf2450d0c97cf82deba053eb218 \
-                    size    91783332
+checksums           rmd160  d05ec291080c000a7f1796b361dc7e957d420c14 \
+                    sha1    f1406235061c8bde8f30a7ceef24ffd7e972b896 \
```

Comment:
`sha1` can be dropped.  Checksums should have `rmd160`, `sha256`, and `size`.

Reply:
@herbygillot here it is!